espaloma/graphs/deploy.py

```diff
@@ -88,6 +88,17 @@ def openmm_system_from_graph(
         sys = ff.create_openmm_system(
             g.mol.to_topology(), charge_from_molecules=[g.mol]
         )
+        
+    elif charge_method == "am1-bcc":
+        g.mol.assign_partial_charges()
+        sys = ff.create_openmm_system(
+            g.mol.to_topology(), charge_from_molecules=[g.mol]
+        )
+        
+    elif charge_method == "keep":
+        sys = ff.create_openmm_system(
+            g.mol.to_topology(), charge_from_molecules=[g.mol]
```

Comment:
Here, you *do*` want `charge_from_molecules`, which will override the charging scheme and take the charges from `g.mol`.

Reply:
Yes, if this is specified, we are assuming the molecule already has charges.